scripts/find-mem-leak.cocci

```diff
@@ -2,8 +2,8 @@
 expression x, y;
 statement S;
 @@
-  x = mbedtls_malloc(...);
-  y = mbedtls_malloc(...);
+  x = mbedtls_calloc(...);
+  y = mbedtls_calloc(...);
   ...
```

Comment:
@andresag01 Thanks! I was wondering because this script would also flag valid, non-leaking code paths which call `free` on both `x` and `y` inside the check, and I think we're using this construct in several places in the library.